tests/managers_regress/models.py

```diff
@@ -147,3 +147,83 @@ class RelationModel(models.Model):
 
     def __str__(self):
         return force_text(self.pk)
+
+
+@python_2_unicode_compatible
```

Comment:
Since you haven't defined any `__str__()` methods, you can remove `@python_2_unicode_compatible` to fix the tests on Python 2.